docker/depth_processor/process_depth.py

```diff
@@ -54,6 +87,7 @@ def main(image_dir, output_dir):
     parser = argparse.ArgumentParser("Depth extraction", add_help=True)
     parser.add_argument("--image_dir", type=str, required=True, help="path to image directory")
```

Comment:
Should we consider making this optional? We could test that at least one flag is being used, otherwise fallback to using an image directory

Reply:
yes, right.